src/pages/adminManagement/semester-management/SemesterListModal.tsx

```diff
@@ -0,0 +1,64 @@
+import arrowIosBackFill from '@iconify/icons-eva/arrow-ios-back-fill';
+import arrowIosForwardFill from '@iconify/icons-eva/arrow-ios-forward-fill';
+import { Icon } from '@iconify/react';
+import { Box, Grid, Typography } from '@material-ui/core';
+import ActionModal from 'components/modal/ActionModal';
+import _ from 'lodash';
```

Comment:
it could be better if we just import directly what we need:
`import map from "lodash/map"`